### DISCUSSION TITLE
Will this project be supporting Python 3.11 any time soon?

### DISCUSSION THREAD

**Discussion by ayushiagrawalofficial** [amoghmadan/Table2XSV#10] [Q&A] [answered]
Now that I know this project is actively maintained ([reference](https://github.com/amoghmadan/Table2XSV/discussions/9)) will there be a support for Python version 3.11 any time soon?

**Comment by amoghmadan** [comment, marked as answer]
Yes there is work which is put into upgrade for Python 3.11 [(reference)](https://github.com/amoghmadan/Table2XSV/tree/feature/move-from-setup-to-pyproject). You would see it in a release somewhere around end of August.

**Reply by ayushiagrawalofficial** [reply]
Thank you
